internal/envelope/envelope.go

```diff
@@ -0,0 +1,10 @@
+package envelope
+
+import (
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+// Payload describes the content that gets signed.
+type Payload struct {
+	TargetArtifact ocispec.Descriptor `json:"targetArtifact"`
```

Comment:
ocispec.Descriptor has more fields than our (signature spec)[https://github.com/notaryproject/notaryproject/blob/main/specs/signature-specification.md#payload]. Is it expected?